packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -321,3 +347,487 @@ describe('Table Column Sizing', () => {
         }
     });
 });
+
+describe('Table Interactive Column Sizing', () => {
+    let element: Table<SimpleTableRecord>;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+    let pageObject: TablePageObject<SimpleTableRecord>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setupInteractiveTests());
+        pageObject = new TablePageObject<SimpleTableRecord>(element);
+        await connect();
+        await element.setData(simpleTableData);
+        await waitForUpdatesAsync();
+        await pageObject.sizeTableToGivenRowWidth(400, element);
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    describe('No hidden columns ', () => {
+        const columnSizeTests = [
+            {
+                name: 'sizing left only affects adjacent right column with delta less than min width',
```

Comment:
The test names and drag deltas here don't seem to match. For example, this test claims to be "sizing left" but provides a positive drag delta.